presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -19,6 +19,41 @@ Operator Description
 ``!=``   Not equal (non-standard but popular syntax)
 ======== ===========
 
+.. _like_operator:
+
+Pattern Comparison: LIKE
+------------------------
+
+The ``LIKE`` operator can be used to compare values with a pattern in a
+``WHERE`` condition::
+
+    ... WHERE column [NOT] LIKE 'pattern' ESCAPE 'character';
+
+Matching characters is case sensitive, and the pattern supports two symbols for
+matching:
+
+- ``_`` denotes matching any single character
+- ``%`` denotes matching zero or more characters
+
+A query to find all regions starting with A, returns AFRICA, AMERICA and ASIA::
+
+    SELECT name FROM tpch.tiny.region WHERE name LIKE 'A%';
+
+You can negate the result by adding ``NOT``, and get EUROPE and MIDDLE EAST::
+
+    SELECT name FROM tpch.tiny.region WHERE name LIKE 'A%';
+
+If you only have one specific character to match, you can use the ``_`` symbol
+for each character. The following query uses two underscores and you receive
+ASIA as result::
+
+    SELECT name FROM tpch.tiny.region WHERE name LIKE 'A__A';
+
+Some characters like ``_`` need to be escaped to allow you to match them. This
+can be achieved by specifying the ``ESCAPE`` character to use::
+
+    SELECT 'a_c' LIKE 'a\_c' ESCAPE '\';
+
```

Comment:
We should mention the escape character can also be escaped.

> To match the escape character, escape it by preceding it with itself.

Reply:
Done